homeassistant/components/rachio/device.py

```diff
@@ -174,6 +184,11 @@ def _setup(self, hass: HomeAssistant) -> None:
             rachio_iro.setup()
             self._controllers.append(rachio_iro)
 
+        base_count = len(base_stations)
+        for base in base_stations:
+            coordinator = RachioUpdateCoordinator(hass, rachio, base, base_count)
+            self._base_stations.append(RachioBaseStation(rachio, base, coordinator))
```

Comment:
This could be `base_stations.extend` with a generator expression

Reply:
Make sure I didn’t screw that up. It works but looks odd to me.